skrypty/inventory/special_inventory.lua

```diff
@@ -1,5 +1,8 @@
 scripts.inv.get_magics_to_put_down_exempts = {
     ["plytach twojego starozytnego pancerza"] = "starozytna runiczna zbroje plytowa",
+    ["mosiezna duza broszka w ksztalcie liscia debu"] = "mosiezna duza broszke w ksztalcie liscia debu",
+    ["zielonym luskowatym plaszczem"] = "zielony luskowaty plaszcz",
+    ["szmaragdowozielonym misternym plaszczem"] = "szmaragdowozielony misterny plaszcz",
```

Comment:
Te chyba nie sa rozpalane?

Reply:
ale jest taka odmiana jak się jest otulonym.
Są dwa rodzaje wyjątków nie ściągane - false, a drugi to jest jakaś inna odmiana `i`